spec.bs

```diff
@@ -3350,38 +3710,78 @@ An auction config is a [=struct=] with the following items:
 : <dfn>seller currency</dfn>
 :: A [=currency tag=]. Specifies the currency bids returned by `scoreAd()` are expected to use, and
   which reporting for this auction will agree on.
-
 : <dfn>per buyer currencies</dfn>
 :: A {{Promise}} or failure or an [=ordered map=] whose [=map/keys=] are [=origins=] and whose
   [=map/values=] are [=currency tags=]. Specifies the currency bids returned by `generateBid()` or
   `scoreAd()` in component auctions are expected to use. The initial value is an empty map.
-
 : <dfn>all buyers currency</dfn>
 :: A [=currency tag=]. Specifies the currency bids returned by `generateBid()` or `scoreAd()` in
   component auctions are expected to use if [=auction config/per buyer currencies=] does not specify
   a particular value.
+: <dfn>auction nonce</dfn>
+:: Null or a [=string=], initially null.

```

Comment:
Can we make this Null or a UUID (and link to the UUID spec that you link to elsewhere).